ast2vec/tests/test_source.py

```diff
@@ -0,0 +1,27 @@
+import os
+import unittest
+
+import ast2vec.tests as tests
+import ast2vec.tests.models as paths
+from ast2vec import Source
+
+
+class SourceTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        tests.setup()
```

Comment:
You do not need this here.

Instead, write `setUp()` and initialize `self.model = Source(source=path)`